nipype/algorithms/tests/test_modelgen.py

```diff
@@ -12,10 +12,18 @@
 import pytest
 import numpy.testing as npt
 from nipype.interfaces.base import Bunch, TraitError
-from nipype.algorithms.modelgen import (SpecifyModel, SpecifySparseModel,
-                                        SpecifySPMModel)
+from nipype.algorithms.modelgen import (bids_gen_info, SpecifyModel,
+                                        SpecifySparseModel, SpecifySPMModel)
 
 
+def test_bids_gen_info():
+    fname = os.path.realpath('data/events.tsv')
```

Comment:
```suggestion
    fname = npt.example_data('events.tsv')
```

Reply:
sorry i forgot that location is not a default in the regular tests.

Reply:
@satra still not working :-/